src/gpodder/sync.py

```diff
@@ -130,14 +130,16 @@ def get_track_length(filename):
             return int(float(mplayer_output[mplayer_output.index('ID_LENGTH'):].splitlines()[0][10:]) * 1000)
         except:
             pass
-    else:
-        logger.info('Please install MPlayer for track length detection.')
 
-    try:
-        mp3file = eyed3.mp3.Mp3AudioFile(filename)
-        return int(mp3file.info.time_secs * 1000)
-    except Exception as e:
-        logger.warn('Could not determine length: %s', filename, exc_info=True)
+    if eyd3mp3_available:
+        try:
+            mp3file = eyed3.mp3.Mp3AudioFile(filename)
+            return int(mp3file.info.time_secs * 1000)
+        except:
+            pass
+
+    logger.warn('Could not determine length: %s', filename)
```

Comment:
those 2 lines would be in an `else:` case

Reply:
Which two lines?